NineChronicles.Headless/HostBuilderExtensions.cs

```diff
@@ -93,45 +93,11 @@ RpcNodeServiceProperties properties
                 {
                     hostBuilder.ConfigureKestrel(options =>
                     {
-                        if (properties.HttpOptions is { } httpOptions)
-                        {
-                            options.ListenAnyIP(httpOptions.Port, listenOptions =>
-                            {
-                                listenOptions.Protocols = HttpProtocols.Http1;
-                            });   
-                        }
```

Comment:
i think we miss this option. is it tested?

Reply:
I missed it but I didn't realize the point 🙏🏻 I tested it through accessing the `/swagger` endpoint. Maybe I guess that it is covered by the lines:

```csharp
options.ListenAnyIP((int)listenPort!, listenOptions =>
                        {
                            listenOptions.Protocols = HttpProtocols.Http1AndHttp2;
                        });
```